### PR TITLE
AP Digital Testing Portal 1:1 Mockup

### PR DESCRIPTION
**Pull request by sresri2**
I've tried to match the testing portal to the AP testing portal (Bluebook Digital Testing), so our portal is a 1:1 of the real digital testing portal. It uses a similar styling to the official testing portal, and I've tried to format everything the same. Some of the main functionality is implemented as well, like the two panels, the information rendering on the left and the question on the right. MCQ formatting is similar to the real testing portal. Navigation between questions is also functional. 
Screenshot 2024-07-21 at 12 18 55 AM
There are some things to work on. **This is just a starting point**. Some of the core functionality is implemented. There is formatting and styling that needs to be fixed/finished. We need to make the articles and questions dynamic by pulling from the database (currently hardcoded in JSON, so needs to be set to our practice questions from the database). 

**tldr: This is a starting point for our 1:1 of the AP Digital Testing Portal**

### CODE REVIEW

File: src/app/digital-testing/_components/ToolsDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// src/app/digital-testing/_components/ToolsDropdown.tsx
+import React, { useState } from 'react';
+import styles from '../styles/ToolsDropdown.module.css';
+import { FaEllipsisV } from 'react-icons/fa'; // Assuming you have react-icons installed
```
[the remark this answers is not in the document]
Comment (alexng353) [reply src/app/digital-testing/_components/ToolsDropdown.tsx:4]: Also I'm not sure what icon library we're using. Ask eton or something.

**Review by alexng353 — COMMENTED** [review]
overall you just need to remove those filename comments (not standardized in our repo, so don't do it)
